src/movement/Pathing.ts

```diff
@@ -47,14 +48,20 @@ export class Pathing {
 		if (!room) {
 			return;
 		}
-		if (room.controller) {
-			if (room.controller.owner && !room.controller.my && room.towers.length > 0) {
-				room.memory[_RM.AVOID] = true;
-			} else {
-				delete room.memory[_RM.AVOID];
-				// if (room.memory.expansionData == false) delete room.memory.expansionData;
-			}
+		if (!(room.controller && room.controller.my) && room.towers.find(tower => tower.isActive()) != undefined) {
```

Comment:
I did find that unclaimed rooms were triggering pathing issues, but yeah cpu is kinda king.
We could make it check if the room controller is owned before caring, instead of just checking it's not owned by you.

HOWEVER: this is an issue because of strongholds. They laser down creeps, and they don't own the room. So we could consider putting a check if "room is not mine and tower is owned by Invader OR current room owner" with find first instead of looking at all.